File: PHP/UberBlack.php
<?php

require_once('Car.php');

class UberBlack extends Car {
    public $typeCarAccepted;
    public $SeatsMaterial;
    

    public function __construct($license,$driver,$typeCarAccepted,$SeatsMaterial){
        parent::__construct($license,$driver)
        $this->typeCarAccepted= $typeCarAccepted;
        $this->SeatsMaterial = $SeatsMaterial;
    }
}

File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    SeatsMaterial = []

    def __init__(self, license, driver,typeCarAccepted,SeatsMaterial) -> None:
        super.__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.SeatsMaterial = SeatsMaterial